doc/whats_new/v1.2.rst

```diff
@@ -49,6 +49,32 @@ Changes impacting all modules
   second-pass algorithm.
   :pr:`23197` by :user:`Meekail Zain <micky774>`
 
+- |Enhancement| Support for combinations of dense and sparse datasets pairs
+  for all distance metrics has been added on the following estimators:
+
+  - :func:`sklearn.metrics.pairwise_distances_argmin`
+  - :func:`sklearn.metrics.pairwise_distances_argmin_min`
+  - :class:`sklearn.cluster.AffinityPropagation`
+  - :class:`sklearn.cluster.Birch`
+  - :class:`sklearn.cluster.MeanShift`
+  - :class:`sklearn.cluster.OPTICS`
+  - :class:`sklearn.cluster.SpectralClustering`
+  - :func:`sklearn.feature_selection.mutual_info_regression`
+  - :class:`sklearn.neighbors.KNeighborsClassifier`
+  - :class:`sklearn.neighbors.KNeighborsRegressor`
+  - :class:`sklearn.neighbors.RadiusNeighborsClassifier`
+  - :class:`sklearn.neighbors.RadiusNeighborsRegressor`
+  - :class:`sklearn.neighbors.LocalOutlierFactor`
+  - :class:`sklearn.neighbors.NearestNeighbors`
+  - :class:`sklearn.manifold.Isomap`
+  - :class:`sklearn.manifold.LocallyLinearEmbedding`
+  - :class:`sklearn.manifold.TSNE`
+  - :func:`sklearn.manifold.trustworthiness`
+  - :class:`sklearn.semi_supervised.LabelPropagation`
+  - :class:`sklearn.semi_supervised.LabelSpreading`
```

Comment:
Have you checked that all those estimators accept sparse inputs?

Reply:
Good catch! I still need to verify that.

Reply:
I removed the unrelated estimator (the ones who do not support sparse data) in the change-log entry via 4c455ea440839eedd6f4b681d0b6eaa636aee4f4.